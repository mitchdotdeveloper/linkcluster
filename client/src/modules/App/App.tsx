export const App = () => <h1>hi</h1>;
