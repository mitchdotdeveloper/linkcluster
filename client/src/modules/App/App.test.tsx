import React from 'react';
import { App } from './App';
import { render } from '@testing-library/react';

describe('App test suite', () => {
  it('App renders correctly', () => {
    const { asFragment } = render(<App />);

    expect(asFragment).toMatchSnapshot();
  });
});
