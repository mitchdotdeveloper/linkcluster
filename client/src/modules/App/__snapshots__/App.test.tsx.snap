// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test suite App renders correctly 1`] = `[Function]`;
